packages/wordcount/src/stripConnectors.js

```diff
@@ -1,14 +1,11 @@
 /**
  * Replaces items matched in the regex with spaces.
  *
- * @param {Object} settings The main settings object containing regular expressions
- * @param {string} text     The string being counted.
+ * @param {import('./index').WPWordCountSettings} settings The main settings object containing regular expressions
+ * @param {string}                                text     The string being counted.
  *
  * @return {string} The manipulated text.
  */
 export default function stripConnectors( settings, text ) {
-	if ( settings.connectorRegExp ) {
-		return text.replace( settings.connectorRegExp, ' ' );
-	}
-	return text;
```

Comment:
@sirreal Is it possible that we botched the rebase? Seems like we lost that conditional (and I'm not sure we're comfortable relying _just_ on TS to make sure the `connectorRegExp` attr is there, since this will continue to be used from JS...)

Same for all the other `packages/wordcount/src/stripSomethings.js`

Reply:
I'll file a commit to fix.

Reply:
Ohh, nevermind, it wasn't us -- it was a deliberate decision to change these. Commit https://github.com/WordPress/gutenberg/pull/22077/commits/2dc5c11cddb60b9af93750460420b535e761e7be has more information.